### ISSUE TITLE
Using target_embed_file triggers "You have to enable ASM" error

### ISSUE THREAD

**Issue by neobrain** [Lectem/3ds-cmake#7]
Embedding files into a binary has nothing to do with shader stuff, so it shouldn't trigger this error either.

Full error message:

```
CMake Error at cmake/Tools3DS.cmake:406 (message):
  You have to enable ASM in order to use add_shader_library.  Use
  enable_language(ASM).  Currently enabled languages are C;CXX
```

**Comment by neobrain** [comment]
Erm, seems like I misunderstood this error message. While the error message mentions `add_shader_library`, the `enable_language(ASM)` thing is probably not related to ASM at all.

If I'm not mistaken, any chance for clearing up that somewhat confusing error message?

**Comment by Lectem** [comment]
oh right, it should be add_binary_library, I probably forgot to rename it. should be fixed by 6d0c65d
